weblate/machinery/microsoft.py

```diff
@@ -28,7 +28,7 @@
 COGNITIVE_BASE_URL = 'https://api.cognitive.microsoft.com/sts/v1.0'
 COGNITIVE_TOKEN = COGNITIVE_BASE_URL + '/issueToken?Subscription-Key={0}'
 
-BASE_URL = 'https://api.microsofttranslator.com/V2/Ajax.svc/'
+BASE_URL = 'https://api.cognitive.microsofttranslator.com/translate?api-version=3.0/'
```

Comment:
This looks wrong, check TRANSLATE_URL and LIST_URL below - these are called for individual API calls. Also, the trailing / looks wrong as it would be parsed as part of the parameter.

You should test this against live service, they have free tier for translate service, which is good enough for testing.